Symfony/CS/Fixer.php

```diff
@@ -83,7 +83,10 @@ public function registerBuiltInConfigs()
         foreach (Finder::create()->files()->in(__DIR__.'/Config') as $file) {
             $relativeNamespace = $file->getRelativePath();
             $class = 'Symfony\\CS\\Config\\'.($relativeNamespace ? $relativeNamespace.'\\' : '').$file->getBasename('.php');
-            $this->addConfig(new $class());
+            $config = new $class();
+            if (!$config instanceof \Exception) {
```

Comment:
`Symfony/CS/Config/InvalidFixerConfigurationException.php`
is now loaded by this custom kind of class loader which expects implementations of 'ConfigInterface` and not `Exception`.....